src/Infrastructure.VS.UnitTests/SolutionInfoProviderTests.cs

```diff
@@ -65,7 +65,7 @@ public void Ctor_DoesNotCallServices()
         [TestMethod]
         [DataRow(null)]
         [DataRow("a value")]
-        public async Task GetAsync_ReturnsExpectedValue(string solutionNameToReturn)
+        public async Task GetSlnFilePathAsync_ReturnsExpectedValue(string solutionNameToReturn)
```

Comment:
Renamed the existing test methods to make it clear which methods they relate to.